src/lib/utils/meta_table_manager.hpp

```diff
@@ -37,11 +37,13 @@ class MetaTableManager : public Noncopyable {
 
  protected:
   friend class Hyrise;
-
   MetaTableManager();
 
+ private:
   static std::string _trim_table_name(const std::string& table_name);
 
+  static std::vector<std::vector<AllTypeVariant>> _materialize_values(const std::shared_ptr<const Table>& values);
```

Comment:
Why not in an anonymous namespace?

Reply:
I also moved `_trim_table_name` to an anonymous namespace.